storybook/src/components/card-grid/card-grid.content.ts

```diff
@@ -6,30 +6,36 @@ export const cards: Card<TemplateResult>[] = [
     href: "#",
     label: "Begrippen uit de Omgevingswet",
     content: html`<p>Begrippen en definities die binnen de Omgevingswet worden gebruikt.</p>`,
+    dsoCardClick() {},
```

Comment:
De events zijn om deze reden altijd optioneel: Er hoeft niet naar ze te worden geluisterd.

Iets wat de moderne ouder ook wel eens met z'n kind mag doen.

Reply:
Is weg, want optioneel in model